### DISCUSSION TITLE
Exclusão de repositório

### DISCUSSION THREAD

**Discussion by ghost** [community/community#114820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Alguém sabe me informar porque eu não consigo excluir um repositório meu? Eu já fiz de tudo e não dá certo, toda vez que digito a frase não dá para clicar no botão de excluir, seja para excluir ou arquivar o repositório.

**Comment by Akash1134** [comment]
Hi @Eduardoteck , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
